pages/osx/nettop.md

```diff
@@ -0,0 +1,35 @@
+# nettop
+
+> Display dynamic real-time information about the network.
+
+- Start nettop monitoring TCP and UDP sockets from all interfaces:
+
+`nettop`
+
+- Start nettop monitoring TCP sockets from Loopback interfaces:
+
+`nettop -m tcp -t loopback`
+
+- Start nettop monitoring specific process:
+
+`nettop -p {{process_id|process_name}}`
+
+- Start nettop displaying summary only for per-process:
+
+`nettop -P`
+
+- Start nettop in logging mode and display 10 samples:
+
+`nettop -l {{10}}`
+
+- Start nettop in delta mode, updating every 5 seconds:
+
+`nettop -d -s {{5}}`
+
+- See interactive commands while running nettop:
```

Comment:
```suggestion
- List interactive commands while running nettop:
```
Not obligatory

Reply:
Probably the semantics of list is clearer, I will do as you say.

Reply:
You could also use `Display` or `Show` but `See` sounds a bit weird in my opinion.